proc/read.go

```diff
@@ -313,6 +316,21 @@ func (p *proccache) getCgroups() ([]procfs.Cgroup, error) {
 	return p.cgroups, nil
 }
 
+func (p *proccache) getCwd() (string, error) {
+	if p.cwd == "" {
+		cwd, err := p.Cwd()
+
+		if err != nil && strings.Contains(err.Error(), "permission denied") {
```

Comment:
Any reason you can't use `errors.Is(err, fs.ErrPermission)`?

Reply:
Hello @ncabatoff,

You are right, I modified the code.